bindings/rust/integration/benches/s2nc.rs

```diff
@@ -2,20 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use criterion::{criterion_group, criterion_main, Criterion};
-use std::{dbg, env, process::Command, time::Duration};
-mod utils;
+use std::{env, process::Command, time::Duration};
 
 pub fn s2nc(c: &mut Criterion) {
     let mut group = c.benchmark_group("s2nc");
     let s2nc_env: &str = &env::var("S2NC_ARGS").unwrap();
-    let s2nc_args: utils::Arguments = s2nc_env.into();
-    let test_name = format!("s2nc_{}", s2nc_args.get_endpoint().unwrap());
-    dbg!("Parsed test_name as: {:?}", &test_name);
-    let s2nc_env: &str = &env::var("S2NC_ARGS").unwrap();
-    let s2nc_args: utils::Arguments = s2nc_env.into();
+    let s2nc_split = s2nc_env.split(' ').collect::<Vec<&str>>();
+    let test_name = format!("s2nc_{}_{}", s2nc_split[5], s2nc_split[6]);
     group.bench_function(test_name, move |b| {
         b.iter(|| {
-            let s2nc_argvec = s2nc_args.clone().get_vec();
+            let s2nc_argvec = s2nc_split.clone();
```

Comment:
This is hard to picture. Can you add a comment with an example of what we expect S2NC_ARGS to look like? Where is S2NC_ARGS going to be set? Do we need to check that it is set / has the right number of arguments?

Reply:
Added a sample and a comment to find this in tests/integrationv2/providers.py#379 (after the PRs are merged) .  The provider is creating these arguments, so if they're wrong, our regular tests would also fail when s2nc  errors out.

Reply:
What would be the `test_name` in this example? `s2nc_test_all_tls13_www.netflix.com`? Wondering if providers.py should just specify the test name so its less fragile

Reply:
@WesleyRosenblum correct on the test_name.  The issue the environment variables work around is we couldn't find a way to get arguments into the harness.  Let me split test_name out into another env variable and set that in providers.py.